<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="download">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>356</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>사진 저장</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="engine_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>검색 엔진</string>
   </property>
   <widget class="QRadioButton" name="google">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Google</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="naver">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Naver</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="github">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="copyright_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Google 이미지 사용권</string>
   </property>
   <widget class="QRadioButton" name="google_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>라이선스로 필터링 안함</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="google_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>수정 후 재사용 가능</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="google_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>재사용 가능</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="google_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>수정 후 비상업적 용도로 재사용 가능</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="google_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>비상업적 용도로 재사용 가능</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="select_folder">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>저장 경로</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="directory_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="cnt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="driver_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_driver">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>driver 경로</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
